# ==============================================================================
# 描述: TeamMulticast消息中，“队友信息表格”的一行。
#      代表关于单个队友的融合情报。
# ==============================================================================
int32 id                            # 队友ID
int32 target_id                     # 已知该队友正在攻击的目标ID (-1表示无目标)
builtin_interfaces/Time timestamp   # 该情报的时间戳